src/fs/fs.go

```diff
@@ -90,8 +90,10 @@ func WriteFile(fromFile io.Reader, to string, mode os.FileMode) error {
 		return err
 	}
 	dir, file := path.Split(to)
-	if err := os.MkdirAll(dir, DirPermissions); err != nil {
-		return err
+	if dir != "" {
```

Comment:
Interesting that filepath.Dir("thing") returns "." whereas this returns ""

Reply:
Yeah, it's strange, but at this point the value of `to` is actually just `filename` (given the command invoked is `plz export outputs -o . :filename`). I think `path.Split` would return `.` for `./filename`.

This is why I'm not convinced this is the perfect solution